pages/decision-reviews/higher-level-review.md

```diff
@@ -104,14 +139,35 @@ If you have evidence to submit, please select another review option.
 
 <li class="process-step list-four">
 
-### Apply
+### Mail, fax, or drop off the form
 
 Fill out the Decision Review Request: Higher-Level Review (VA Form 20-0996). <br>
 <a href="https://www.vba.va.gov/pubs/forms/VBA-20-0996-ARE.pdf">Download VA Form 20-0996 (PDF)</a>
 
+You can also ask for a copy of this form at a regional office, or by calling the VA toll-free hotline, 800-827-1000, Monday through Friday, 8:00 a.m. to 9:00 p.m.
```

Comment:
I'm not sure if this applies here, but in HLR's footer, we're using the MYVA411 number which has the hours of 24/7 - see https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/disability-benefits/996/content/GetFormHelp.jsx#L11